### PR TITLE
Right click context menu on links - DES-10489

### PR DESCRIPTION
**Pull request by rbarros-daitan**
Coding changes for DES-10489.

### CODE REVIEW

**Comment by rbarros-daitan** [comment]
@lneir , here is PR for Context Menu Links of DES-10489

File: minuet/Paragon.Plugins.Notifications/Paragon.Plugins.Notifications.csproj [hunk 1]
```diff
@@ -1,179 +1,179 @@
...
-      <Name>Paragon.Plugins.MessageBus</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\Paragon.Runtime\Paragon.Runtime.csproj">
-      <Project>{766A5435-5221-40FF-825E-BE001A7D6662}</Project>
-      <Name>Paragon.Runtime</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup />
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (lneir) [inline minuet/Paragon.Plugins.Notifications/Paragon.Plugins.Notifications.csproj]: what's going on here.  looks like nothing changed in this file??? please clean up.

**Comment by lneir** [comment]
i tested code and don't see inspect.  please add.

**Comment by lneir** [comment]
lgtm

**Pull request merged**